packages/unlock-js/src/walletService.ts

```diff
@@ -402,4 +402,25 @@ export default class WalletService extends UnlockService {
     }
     return version.setExpirationDuration.bind(this)(params, callback)
   }
+
+  async getCancelAndRefundValueFor(
+    params: {
+      lockAddress: string
+      owner: string
+    },
```

Comment:
the token address seems to be missing even though it is used in the gasRefundFunction above. 

Add the declaration for its parameters, please.